Mathlib/Algebra/Homology/Embedding/StupidTrunc.lean

```diff
@@ -0,0 +1,110 @@
+/-
+Copyright (c) 2024 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou
+-/
+
+import Mathlib.Algebra.Homology.Embedding.Extend
+import Mathlib.Algebra.Homology.Embedding.IsSupported
+import Mathlib.Algebra.Homology.Embedding.Restriction
+
+/-!
+# The stupid truncation of homological complexes
+
+Given an embedding `e : c.Embedding c'` of complex shapes, we define
+a functor `stupidTruncFunctor : HomologicalComplex C c' ⥤ HomologicalComplex C c'`
+which sends `K` to `K.stupidTrunc e` which is defined as `(K.restriction e).extend e`.
```

Comment:
I think this means that one of the TODO's in `Mathlib/Algebra/Homology/Embedding/Basic.lean` is now done, on line 33/34.

Reply:
Thanks!